app/subscriber/src/features/my-reports/edit/ReportEditActions.tsx

```diff
@@ -93,13 +93,7 @@ export const ReportEditActions = ({ disabled, onGenerate, onPublish }: IReportEd
           <FaSave />
         </Button>
       </Show>
-      <Show
-        visible={
-          !!instance?.sentOn &&
-          !active?.startsWith(ReportMainMenuOption.Settings) &&
-          getReportKind(values) === ReportKindName.Manual
-        }
-      >
+      <Show visible={!!instance?.sentOn && !active?.startsWith(ReportMainMenuOption.Settings)}>
```

Comment:
Allow starting the next report on an Auto Report.  I think this is an odd use-case as there isn't much point in an Auto Report if it's started manually.